script/release-timestamp

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# A script to update the timestamp of a Flynn TUF repository
+
+set -eo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+source "${ROOT}/script/lib/aws.sh"
+source "${ROOT}/script/lib/tuf.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [options]
+
+OPTIONS:
+  -h            Show this message
+  -b BUCKET     The S3 bucket containing the TUF repository [default: flynn]
+  -d DIR        Path to the local TUF repository [default: /etc/flynn/tuf]
+
+Requires AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY to be set
+USAGE
+}
+
+main() {
+  local bucket tuf_dir
+
+  while getopts "hb:d:" opt; do
+    case $opt in
+      h)
+        usage
+        exit 1
+        ;;
+      b) bucket=${OPTARG} ;;
+      d)
+        tuf_dir=${OPTARG}
+        if [[ ! -d "${tuf_dir}" ]]; then
+          fail "No such directory: ${tuf_dir}"
+        fi
+        ;;
+      ?)
+        usage
+        exit 1
+        ;;
+    esac
+  done
+  shift $((${OPTIND} - 1))
+
+  if [[ $# -ne 0 ]]; then
+    usage
+    exit 1
+  fi
+
+  info "downloading existing TUF metadata"
+  download_tuf_metadata "${tuf_dir}" "${bucket}"
+
+  cd "${tuf_dir}"
+
+  info "checking snapshot expires"
+  if metadata_expired "repository/snapshot.json"; then
```

Comment:
We should check if it expires in the next N, where N is the next two or three check intervals. If it's expired, we're too late.